src/u2f/u2f_app.c

```diff
@@ -16,6 +17,26 @@ typedef struct {
     const char* name;
 } app_t;
 
+// shorten (and split as array) menonic for displaying hashes in a more recognizable way, at the
+// cost of less entropy, output configurable for other use return success
+static bool _short_mnemonic(char* mnemonic, const char** wordlist_out, uint8_t num_words)
+{
+    // break out if the target array is too short
+    if (num_words > sizeof(wordlist_out)) {
```

Comment:
unfortunately C strikes again: sizeof(wordlist_out) is only the size of a pointer, which is 4 or 8 :see_no_evil:  to make it easy, you can just drop this whole check.

Reply:
c sux. done.